### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by lduffy-voysis**
Updated the readme to include information on the config file and the distinction between our rest and websocket service.

Note: I still have to add the optional userId field on android.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,9 +28,29 @@ Overview
...
+- **isVadEnabled** - *Boolean:* `VAD` stands for Voice Activation Detection. 
+If set to true the library will automatically detect when the user has stopped speaking and process requests. 
+This is made possible via an `okHttp` Websocket connection. See [Websocket docs](https://developers.voysis.com/docs/websocket-api) for more info on our Websocket api.
+If turned off the library will default to a REST interface and the user will need to manually call `service.finish()` to process requests. See [REST docs](https://developers.voysis.com/docs/rest-api) for more info on our REST api.
+*NOTE*: It is preferable to have `isVadEnabled` set to `true` as the Websocket provides the user with more functionallity and flexability.
+
+- **url** - *URL:* This is the endpoint that network requests will be executed against. 
+The url is provided when your Voysis service is delivered. To sign up for a Voysis service visit our [homepage](https://voysis.com/)   
+
+- **refreshToken** - *String:* The refresh token is used for authenticating unique users and for refreshnig session tokens.
```
Comment (dean-voysis) [inline README.md]: typo in refreshing

File: README.md [hunk 2]
```diff
@@ -28,9 +28,29 @@ Overview
 
 The `Service` class is the main interface used to process voice recognition requests.
 It is accessed via the `ServiceProvider().make(context , config)` method. See demo application for more details.
-The sdk communicates to the network using a websocket connection accomplished using `Okhttp websocket`.
+The sdk uses either a REST or Websocket connection depending on how the config file is setup.
```
[the remark this answers is not in the document]
Comment (lduffy-voysis) [reply README.md]: I added an extra not here reminding users that its preferable to use a websocket.

File: README.md (hunk 1 above)
Comment (lduffy-voysis) [reply README.md]: fixed typo

[the remark this answers is not in the document]
Comment (lduffy-voysis) [reply README.md]: fixed typo

**Pull request merged**